tests/Functional/RepositoryProxyTest.php

```diff
@@ -342,7 +344,9 @@ public function can_use_get_count(): void
         $this->assertSame(4, $categoryFactoryClass::repository()->getCount());
     }
 
+    /** @phpstan-return \Zenstruck\Foundry\Tests\Fixtures\Entity\Category|\Zenstruck\Foundry\Tests\Fixtures\Document\ODMCategory */
     abstract protected function categoryClass(): string;
 
+    /** @phpstan-return \Zenstruck\Foundry\Tests\Fixtures\Factories\CategoryFactory|\Zenstruck\Foundry\Tests\Fixtures\Factories\ODM\CategoryFactory */
     abstract protected function categoryFactoryClass(): string;
```

Comment:
- these methods does not return actual objects but their class, so we can use `class-string` 
- I think there is no need for the `phpstan-` prefix?

```suggestion
    /** @return class-string<ODMCategory|ORMCategory> */
    abstract protected function categoryClass(): string;
    
    /** @return class-string<ODMCategoryFactory|ORMCategoryFactory> */
    abstract protected function categoryFactoryClass(): string;
```